data/mods/Aftershock/npcs/mutant_npcs/notes.md

```diff
@@ -0,0 +1,13 @@
+[
```

Comment:
You probably don't need full `modinfo.json ` from obsolete mod.

Reply:
So just keep the first sentence or delete the whole file?

Reply:
I suggest deleting the whole file.

Reply:
Deleted. Thanks.